packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -61,6 +61,7 @@ export function SignIn() {
             className="amplify-flex"
             disabled={isPending}
           >
+            <legend>Sign in</legend>
```

Comment:
Will this be translated? Also, this changes the design of the Authenticator. Should this be visually hidden @ErikCH @wlee221

Reply:
Yes this will change the design of Authenticator. It would be better if you can ping them to take a look at

Reply:
Does this fix an accessibility issue?

Reply:
Yes, there's an alert that `fieldset` missing legend

Reply:
I forgot to add a `amplify-visually-hidden` class...I'll do that

Reply:
Yep, adding visually-hidden should be sufficient.